manage.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python

```

Comment:
Exclude the `.idea` and `__pyache__` directories with their content from the pull request.